action/output.php

```diff
@@ -54,11 +38,11 @@ public function handle_read(Doku_Event $event, $param) {
      * @param $param
      */
     public function handle_output(Doku_Event $event, $param) {
-        global $ID;
-        if($this->lastread != $ID) return; // avoid nested calls
-        $this->lastread = '';
+        global $ID, $ACT;
+        if (act_clean($ACT) !== 'show') {
+            return;
```

Comment:
Could it happen that the parse step is done during another mode than show? Then the instructions would be missing. I don't think we need this check here, since we're testing during rendering now.